src/NHibernate.Test/Async/CfgTest/SettingsFactoryFixture.cs

```diff
@@ -0,0 +1,82 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by AsyncGenerator.
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+
+using System;
+using System.Collections.Generic;
+using NHibernate.AdoNet;
+using NHibernate.Bytecode;
+using NHibernate.Cache;
+using NHibernate.Cfg;
+using NHibernate.Connection;
+using NHibernate.Exceptions;
+using NHibernate.Hql;
+using NHibernate.Hql.Ast.ANTLR;
+using NHibernate.Linq.Functions;
+using NHibernate.Linq.Visitors;
+using NHibernate.Transaction;
+using NSubstitute;
+using NUnit.Framework;
+using Environment = NHibernate.Cfg.Environment;
+
+namespace NHibernate.Test.CfgTest
+{
+	using System.Threading.Tasks;
+	[TestFixture]
+	public class SettingsFactoryFixtureAsync
```

Comment:
This class should not be generated as there is nothing async, it is a bug related to async generator that I need to fix.

Reply:
I released a new version of the async generator (`0.8.2.10`) that fixes [the issue](https://github.com/maca88/AsyncGenerator/issues/116) of generating non async methods.